azure_translator/__init__.py

```diff
@@ -81,8 +86,7 @@ def translate(self, text, to=DEFAULT_LANGUAGE, source_language=None):
                 response=error.response,
                 request=error.request
             )
-
         try:
-            return ET.fromstring(resp.content).text
-        except ET.ParseError as e:
+            return json.loads(resp.content)[0]['translations'][0]['text']
```

Comment:
use `response.json()` to get the unmarshaled json object.

Reply:
check json is correctly formatted

Reply:
My initial implementation was with .json(), But I had a lot of trouble with the tests with that implementation, plus I'm not sure if it makes a difference here since we only care about returning the 'text' field. @dszczyt pls  clarify is the response.json() is absolutely necessary here.